web-components/tsconfig.json

```diff
@@ -5,22 +5,17 @@
     "declaration": true,
     "jsx": "react-jsx",
     "noEmitOnError": true,
+    "types": ["node", "jest"],
     "outDir": "lib",
     "rootDir": "src",
     "module": "esnext",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
```

Comment:
we keep getting diffs like this, where the same files are moved to a different format without change. Wonder if there’s sometheing off with our IDE settings...